.lando.yml

```diff
@@ -213,10 +211,24 @@ services:
 
   # Any emails generated by Drupal will be caught by this service, and
   # can be seen in the browser.
-  mailhog:
-    type: mailhog
-    hogfrom:
-      - appserver
+  mailpit:
+      type: lando
+      api: 3
+      ssl: true
+      sslExpose: false
+      services:
+        image: axllent/mailpit
+        volumes:
+        - .lando/data-mailpit:/data
+        ports:
+          - "8025"  # Web UI
+          - "1025"  # SMTP
+        environment:
+          MP_MAX_MESSAGES: 5000
+          MP_DATA_FILE: /data/mailpit.db
+          MP_SMTP_AUTH_ACCEPT_ANY: 1
+          MP_SMTP_AUTH_ALLOW_INSECURE: 1
+        command: "/mailpit"
```

Comment:
mailhog is not supported anymore. Mailpit is, and it also integrates nicely with cypress https://www.npmjs.com/package/cypress-mailpit